src/Microsoft.Framework.PackageManager/Restore/NuGet/NonexistentPackageFolder.cs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.Framework.PackageManager.Restore.NuGet
+{
+    public class NonExistentPackageFolder : IPackageFeed
```

Comment:
If the structure of a folder doesn't conform with `NuGetPackageFolder`, we always assume it is a `KpmPackageFolder`. So I remove `NonExistentPackageFolder` and move the logic to `KpmPackageFolder`. Do you like this approach? https://github.com/aspnet/KRuntime/commit/b34cda0ac6924c8e911133bb6198bd36fa89b855